.github/workflows/release.yml

```diff
@@ -0,0 +1,42 @@
+name: Create Release and Tag
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Version for the release"
+        required: true
+        default: "0.1.0"
```

Comment:
please remove this: we should read the version from a config file that is added to the repo.
chatgpt can advise about how to do that with go